client/src/pages/OrganisationProfile.js

```diff
@@ -532,18 +534,17 @@ const OrganisationProfile = ({ isAuthenticated }) => {
 
   useEffect(() => {
     setSwitchOnOff(isJoinOrg);
-    setPosDescription(description);
-  }, [organisation, setSwitchOnOff]); // eslint-disable-line react-hooks/exhaustive-deps
+    //setPosDescription(description)
+  }, [isJoinOrg, organisation, setSwitchOnOff]);
```

Comment:
did you add this dependency here isJoinOrg?
Also add // eslint-disable-line react-hooks/exhaustive-deps otherwise useEffect will add dependencies